modules/apps/site-initializer/site-initializer-raylife-d2c/extra/remote-app/src/routes/get-a-quote/hooks/useFormActionsDefault.js

```diff
@@ -93,6 +96,13 @@ const useFormActions = ({
 
 			setError('continueButton', {});
 
+			if (selectedStep.index === CONTACT_INFORMATION_STEP) {
+				status = APPLICATION_STATUS.OPEN;
+			}
+			else {
+				status = APPLICATION_STATUS.INCOMPLETE;
+			}
+
```

Comment:
Please, change this. Does it make sense for you?

```suggestion
                        status = APPLICATION_STATUS.OPEN;

			if (selectedStep.index !== CONTACT_INFORMATION_STEP) {
				status = APPLICATION_STATUS.INCOMPLETE;INCOMPLETE
			}

```

Reply:
Done!